modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/source/PollingSourceWrapperTestCase.java

```diff
@@ -223,30 +223,6 @@ private void setComponentLocationMock() throws Exception {
     setFieldValue(pollingSourceWrapper, "componentLocation", componentLocationMock, false);
   }
 
-  private Logger createMockLogger() {
-    Logger logger = mock(Logger.class);
-    Answer answer = new Answer() {
-
-      @Override
-      public Object answer(InvocationOnMock invocation) throws Throwable {
-        String method = invocation.getMethod().getName();
-        String message = invocation.getArgument(0, String.class);
-        Object[] messageArgs = Arrays.copyOfRange(invocation.getArguments(), 1, invocation.getArguments().length);
-        if (method.equals("debug")) {
-          debugMessages.add(formatMessage(message, messageArgs));
-        } else {
-          traceMessages.add(formatMessage(message, messageArgs));
-        }
-        return null;
-      }
-    };
-    doAnswer(answer).when(logger).debug(anyString(), (Object) any());
-    doAnswer(answer).when(logger).debug(anyString(), any(), any());
-    doAnswer(answer).when(logger).trace(anyString(), any(), any());
-    doAnswer(answer).when(logger).trace(anyString(), (Object[]) any());
-    return logger;
-  }
-
   private String formatMessage(String message, Object... args) {
```

Comment:
I think that this method could now be removed from this class, given that it is used on the LoggingTestUtils.java class, right?

Reply:
You are right, I am going to remove it.